### PR TITLE
refactor: use jina commons

### PR DESCRIPTION
**Pull request by slettner**
Issue: https://github.com/jina-ai/jina-commons/issues/6

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:2.0.0rc9
+FROM jinaai/jina:2.0
```
Comment (florian-hoenicke) [inline Dockerfile:1]: update jina version

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 sentence-transformers==2.0.0
-jina==2.0.0rc9
+jina[hub]==2.0
```
Comment (florian-hoenicke) [inline requirements.txt:2]: check if needed

File: Dockerfile (hunk 1 above)
Comment (slettner) [reply Dockerfile:1]: Updated in https://github.com/jina-ai/executor-sentence-transformer/pull/3/commits/6f331d2e31b15f73dc955cacdf7161fe8c6d6f5d

File: requirements.txt (hunk 2 above)
Comment (slettner) [reply requirements.txt]: Updated in https://github.com/jina-ai/executor-sentence-transformer/pull/3/commits/6f331d2e31b15f73dc955cacdf7161fe8c6d6f5d

**Review by florian-hoenicke — APPROVED** [review]

**Pull request merged**